SimpleTimer.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using SubsurfaceStudios.Attributes;
+using SubsurfaceStudios.Utilities.Async;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace SubsurfaceStudios.Utilities.Time
+{
+    [System.Serializable]
```

Comment:
This comes down to personal preference, but I would suggest adding a `using System;` at the top of this file and switching this attribute to just `[Serializable]`. Thank you!